provisioning_templates/provision/kickstart_default.erb

```diff
@@ -44,7 +44,19 @@ This template accepts the following parameters:
 install
 <% end -%>
 <% if host_param('kickstart_liveimg') %>
-liveimg --url=<%= host_param('kickstart_liveimg') %> <%= proxy_string %>
+<%
```

Comment:
One nitpick: we currently have:

```ruby
<% if host_param('kickstart_liveimg') %>
<%
img_name = host_param('kickstart_liveimg')
liveimg_url = if host_param('kt_activation_keys')
```

Probably it should be just:

```ruby
<%
if host_param('kickstart_liveimg')
  img_name = host_param('kickstart_liveimg')
  liveimg_url = if host_param('kt_activation_keys')
  # ...
```

Reply:
Fixed